content/blog/esc-imports-visualization/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Pulumi ESC: Visualizing Environment Imports"
+date: 2024-12-13T00:00:00-07:00
+draft: false
+allow_long_title: true
+meta_desc: "Visualizing Pulumi ESC Environment imports empower you to manage and update your secrets and configuration with confidence."
+meta_image: "meta.png"
+authors:
+- sean-yeh
+- arun-loganathan
+tags:
+  - esc
+  - secrets
+  - config management
+  - features
+  - releases
+---
+
+Managing secrets and configuration across multiple environments and stacks can easily become complex, leading to duplicated values, inconsistencies, and security risks. [Pulumi ESC](/product/esc) solves this with composable environments via [imports](/docs/esc/environments/imports/), allowing you to define configuration once and reuse it organization-wide.  Now, with the new visualization of environment imports, you gain unprecedented visibility and control, simplifying the management of even the most complex infrastructure and applications.
+
+<!--more-->
+
+## Understanding Pulumi ESC Imports
+
+Pulumi ESC lets you define secrets and configuration as code collections called [environments](/docs/esc/environments/working-with-environments/). These environments can import other environments, creating a hierarchical structure that promotes reusability, and reduces duplication and sprawl. You can centralize secrets and configuration within ESC and use them across [Pulumi IaC Stacks](/docs/esc/integrations/infrastructure/pulumi-iac/), [Pulumi Insights Accounts](/docs/pulumi-cloud/insights/accounts/) and with various other platforms through our [integrations](/docs/esc/integrations/).
+
+By using [version tags](/docs/esc/environments/versioning/#tagging-versions), imports can target specific versions of an environment. This locks down the configuration used by dependents, ensuring stability and preventing unintended changes rolling out to your critical environments. 
```

Comment:
> imports can target specific versions of an environment.
Not just version tags, but also just by revision? Not sure if it matters to be that specific

Reply:
Yes, I added that before but we would like to specifically encourage people to use tags - so removed that.